index.d.ts

```diff
@@ -15,15 +15,28 @@ declare const domLoaded: Promise<void> & {
 	/**
 	Synchronously check if the DOM has already finished loading.
 
+	@example
 	```
 	import domLoaded from 'dom-loaded';
 
 	if (domLoaded.hasLoaded) {
-		console.log('The DOM has already finished loading.')
+		console.log('The DOM has already finished loading.');
 	}
 	```
 	*/
 	readonly hasLoaded: boolean;
+
+	/**
+	AbortSignal that is aborted when the DOM has already finished loading.
```

Comment:
It's unclear from the description what happens if the DOM has not already finished loading.

Reply:
```suggestion
	An `AbortSignal` that triggers when the DOM finishes loading or immediately if it has already loaded.
```